src/tests/PostTest.test.ts

```diff
@@ -86,11 +87,13 @@ describe('post tests', () => {
       .write();
 
     expectedPost.user = post.user;
+    let newlyExpectedPost = expectedPost as unknown as PostWithIsSaved
+    newlyExpectedPost.isSaved = false
 
-    const getPostResponse = await postController.getPostById(uuidParam);
+    const getPostResponse = await postController.getPostById(post.user, uuidParam);
     getPostResponse.post.price = Number(getPostResponse.post.price);
     expectedPost.created = getPostResponse.post.created;
-    expect(getPostResponse.post).toEqual(expectedPost);
+    expect(getPostResponse.post).toEqual(newlyExpectedPost);
```

Comment:
I think adding a test case for when `isSaved` is `true` would be useful too

Reply:
added